source/locale/pt_BR/LC_MESSAGES/releases.po

```diff
@@ -85,107 +84,110 @@ msgstr ""
 
 #: ../../source/releases.rst:28
 msgid "New node Event Device In"
-msgstr ""
+msgstr "Novo Nó Evento Dispositivo (Entrada)"
 
 #: ../../source/releases.rst:29
+#, fuzzy
 msgid "New node Event Template Device"
-msgstr ""
+msgstr "Novo nó Evento de modelos de dispositivo (Entrada)"
 
 #: ../../source/releases.rst:30
 msgid "New node FTP"
-msgstr ""
+msgstr "Novo nó FTP"
 
 #: ../../source/releases.rst:31
 msgid "New node Multi Device Out"
-msgstr ""
+msgstr "Novo nó de Saída para  vários dispositivos (Multi Device Out)"
 
 #: ../../source/releases.rst:32
-#, fuzzy
 msgid "New node Notification"
-msgstr "Notificações"
+msgstr "Novo nó de Notificações"
 
 #: ../../source/releases.rst:33
 msgid "New node Multi Actuate"
-msgstr ""
+msgstr "Novo nó de multiplias atuações (Multi Actuate)"
 
 #: ../../source/releases.rst:34
 msgid "New node Cron"
-msgstr ""
+msgstr "Novo nó Cron"
 
 #: ../../source/releases.rst:35
 msgid "New node Cron Batch"
-msgstr ""
+msgstr "Novo nó Cron Batch"
 
 #: ../../source/releases.rst:36
 msgid "New node Cumulative Sum"
-msgstr ""
+msgstr "Novo nó de Soma acumulada"
 
 #: ../../source/releases.rst:37
 msgid "New node Merge data"
-msgstr ""
+msgstr "Novo nó de Mesclar dados "
 
 #: ../../source/releases.rst:38
-#, fuzzy
+msgid "Node email has been removed"
+msgstr "Node email has been removed"
+
+#: ../../source/releases.rst:39
 msgid "Internacionalization"
 msgstr "Internacionalização"
 
-#: ../../source/releases.rst:39
+#: ../../source/releases.rst:40
 msgid "Support to handlebars template on template node"
-msgstr ""
+msgstr "Suporte a *handlebars template*  no nó de template"
 
-#: ../../source/releases.rst:41
+#: ../../source/releases.rst:42
 msgid "History:"
-msgstr ""
+msgstr "History:"
 
-#: ../../source/releases.rst:43
+#: ../../source/releases.rst:44
 msgid "New endpoint to query notifications"
 msgstr "Novo endpoint para consultar notificações"
 
-#: ../../source/releases.rst:45
+#: ../../source/releases.rst:46
 msgid "ImageManager:"
-msgstr ""
+msgstr "ImageManager:"
 
-#: ../../source/releases.rst:47
+#: ../../source/releases.rst:48
 msgid "Improvements to support Firmware Update"
 msgstr "Melhorias para dar suporte à atualização de firmware"
 
-#: ../../source/releases.rst:49
+#: ../../source/releases.rst:50
 msgid "DataBroker:"
 msgstr ""
```

Comment:
missing?

Reply:
No, it just doesn't have to be translated.